ansys/dpf/core/path_utilities.py

```diff
@@ -39,6 +40,9 @@ def join(*args, **kwargs):
     for a in args:
         if isinstance(a, str) and len(a) > 0:
```

Comment:
you can make it simpler with:
```
if isinstance(a, (str, Path)) and len(str(a)) > 0:
    parts.append(str(a))
```

Reply:
sorry, didnt ping you when replying @cbellot000

Reply:
or we could do:

```python
if isinstance(a, (str, Path)) and Path(a) != Path(""):
    ...
```

Reply:
@jfthuong works for me! Modifying now.